apps/snitch_api/lib/snitch_api_web/controllers/user_controller.ex

```diff
@@ -54,4 +56,20 @@ defmodule SnitchApiWeb.UserController do
     user = Guardian.Plug.current_resource(conn)
     render(conn, "current_user.json-api", data: user)
   end
+
+  def wishlist_items(conn, %{"id" => id}) do
+    user_id = String.to_integer(id)
+
+    with user <- UserModel.get(user_id) do
```

Comment:
Why user schema needed in son

Reply:
As per json api spec the response for fetching a relationship should be 
```json
HTTP/1.1 200 OK
Content-Type: application/vnd.api+json

{
  "links": {
    "self": "/articles/1/relationships/tags",
    "related": "/articles/1/tags"
  },
  "data": [
    { "type": "tags", "id": "2" },
    { "type": "tags", "id": "3" }
  ],
included: {
  ...
}
}
```

But Ja_serializer doesn't support any thing of this sort. So, kept it this way.